packages/react-composites/tests/browser/call/hermetic/VideoGalleryLayout.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  addVideoStream,
+  buildUrlWithMockAdapter,
+  defaultMockCallAdapterState,
+  defaultMockRemoteParticipant,
+  test
+} from './fixture';
+import { expect } from '@playwright/test';
+import { dataUiId, waitForSelector, stableScreenshot, isTestProfileMobile, pageClick } from '../../common/utils';
+import { IDS } from '../../common/constants';
+
+test.describe('VideoGalleryLayout tests', async () => {
+  /* @conditional-compile-remove(pinned-participants) */
+  test('VideoTile pin/unpin the remote participant', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileMobile(testInfo));
```

Comment:
its not clear from this test that its only meant for desktop besides ignoring this for mobile. 

Should there be a mobile test as well?

Reply:
yes, there are separate tests for mobile drawer menu on long press. Modified the test title to make it clear that it is only for desktop